### DISCUSSION TITLE
How to set a nglview widget in a play mode in a loop

### DISCUSSION THREAD

**Discussion by AndresOrtegaGuerrero** [nglviewer/nglview#1083] [Q&A] [answered]
Is there the possibility of controlling if the widget containing a trajectory 
from nglview can be set to play in a loop by default, without the user having the need to click in the play button?

**Comment by hainm** [comment, marked as answer]
> Is there the possibility of controlling if the widget containing a trajectory from nglview can be set to play in a loop by default, without the user having the need to click in the play button?

@AndresOrtegaGuerrero Please see this thread: https://github.com/nglviewer/nglview/issues/1041

**Reply by AndresOrtegaGuerrero** [reply]
Hello @hainm , yes I am interested into having a control how the widget appears in the jupyter notebook , without having a infinite loop running in the background tried with `ngl._iplayer.children[0]._playing = True` `ngl._iplayer.children[0]._repeat = True` but it doesnt triggers the logic.

**Reply by hainm** [reply]
Hello @AndresOrtegaGuerrero: I will think about this. I don't have an answer currently. Cheers.

Hai
